test/lib/ansible_test/_internal/cli.py

```diff
@@ -527,6 +527,11 @@ def key_value_type(value):  # type: (str) -> t.Tuple[str, str]
     sanity.add_argument('--base-branch',
                         help=argparse.SUPPRESS)
 
+    sanity.add_argument('--validate-deprecation',
+                        action='store_true',
+                        help='Compare deprecation versions to current Ansible '
+                             'version resp. collection version')
+
```

Comment:
Validation should always be enabled. Otherwise the validity of sanity ignores will be dependent on CLI options, which is something we work very hard to avoid.

Reply:
In the meeting, it was explicitly wished that this is not enabled by default.

https://meetbot.fedoraproject.org/ansible-meeting/2020-03-17/ansible_core_public_irc_meeting_https:github.comansiblecommunityissues528.2020-03-17-19.01.log.html
See timestamp 19:34:00 and later.

I personally would like it to be enabled by default as well, but especially when doing deprecations by date (#68177), this will result in situations where tests suddenly fail from one day to the next. I think that's the main motivation to not enable it by default.

Reply:
@mattclay there's currently no always-on deprecation by version check. It is always-off, and the only way to turn it on is to edit ansible-test internals. Or do you mean the current state of this PR?

Reply:
When I looked at the pylint config for ansible (not collections), the `git blame` showed I added the entry to disable `ansible-deprecated-version` and I was thinking I'd added it by mistake when splitting the config between ansible and collections. I see now that wasn't the case.

That being said, the deprecation by version could be always-on since we have ignores. I think it would be more useful as such. For the reasons already stated, the deprecation by date cannot be.